src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { serve } from '@hono/node-server'
+import { Hono } from 'hono'
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+const app = new Hono()
+
+app.get('/users', async (c) => {
+  const allUsers = await prisma.users.findMany()
+  return c.json({ data: allUsers, ok: true })
+})
+
+app.get('/users/:id', async (c) => {
+  const id = Number(c.req.param('id'))
+  const user = await prisma.users.findUnique({
+    where: { id },
+  })
+
+  if (!user) {
+    return c.json({ error: 'Not Found', ok: false, data: user }, 404)
+  }
+
+  return c.json({ data: user, ok: true })
+})
+
+app.post('/users', async (c) => {
+  const { login, email } = c.req.query()
```

Comment:
Please add a validation middleware for the logic and email and user `c.req.valid()`